pixel.js

```diff
@@ -0,0 +1,87 @@
+// create grid; takes two arguments, both numbers are units of pixel boxes
+const grid = document.querySelector('.grid');
+const horizontalPixels = 40;
+const verticalPixels = 60;
+
+function makeGrid() {
+  for (let i = 0; i < verticalPixels; i++) {
+    const row = document.createElement('div');
+    row.classList.add('row');
+
+      for (let j = 0; j < horizontalPixels; j++) {
+        const pixel = document.createElement('div');
+        pixel.classList.add('pixel');
+        row.appendChild(pixel);
+      }
+    grid.appendChild(row);
+  }
+ }
+
+ function clearGrid() {
+   grid.innerHTML = "";
+   makeGrid();
+ }
+
+
+
+makeGrid();
+
+let gridSize = function() {
```

Comment:
you're mixing function expression (right above) with function definitions. Please be consistent.